### ISSUE TITLE
Add content preview panel

### ISSUE THREAD

**Issue by swg0101** [skylot/jadx#1211]
It would be extremely useful if there is a preview panel that allows the preview of code as you are scrolling/moving through the navigation pane. Sometimes when you are just flipping through the classes, you do not actually want to clutter your tabs with things you are not really interested in. Rather, you would keep pressing the down arrow key to allow you to explore through the source code to find interesting code you want to open.

Currently, jadx requires you to actually open the file in order to see it. This is actually somewhat inconvenient when you are just flipping through code since you end up with 20-30 different tabs that are opened and you end up having to close them all back to try to find the source file you originally intended to look at. Having a temporary preview pane alleviates this tab clutter and allows much easier navigation through more complicated code.

**Comment by LBJ-the-GOAT** [comment]
> you end up having to close them all back to try to find the source file you originally intended to look at

Hi, you can press the `ESC` key, it's a shortcut for go back and very useful for a quick glance. 👍

**Comment by swg0101** [comment]
> > you end up having to close them all back to try to find the source file you originally intended to look at
> 
> Hi, you can use the `ESC` key, it's a shortcut for go back and very useful for a quick glance. 👍

Yeah, I actually found that the back and forward mouse buttons actually work in the nightlies as well, which is pretty awesome in my opinion. The problem is just that I still couldn't preview files very efficiently, especially when going through a list of 200 XML files and you need to determine which one has the string you were looking for (but the search resources function doesn't seem to pick these up right now)...

**Comment by MrIkso** [comment]
Samse as #756
@skylot this issue nedd close, it a completed
